packages/hotspot/configure/src/hotspot-container.jsx

```diff
@@ -150,6 +139,11 @@ class Container extends Component {
              } : {}}
         >
           <div className={classes.toolbar}>
+            {imageUrl ? (
+              <div className={classes.replaceSection}>
```

Comment:
As per our style guide - can you favor using jsx instead of calling a function?
```jsx
{ imageUrl && <UploadControl label="Replace Image" className={classes.replaceButton} /> }
//...
```

Reply:
Done